models/sipnet/R/split_inputs.SIPNET.R

```diff
@@ -20,40 +20,44 @@ split_inputs.SIPNET <- function(settings, start.time, stop.time, inputs, overwri
   met <- inputs
   path <- dirname(met)
   prefix <- sub(".clim", "", basename(met), fixed = TRUE)
-  if(is.null(outpath)){
+  if (is.null(outpath)) {
     outpath <- path
   }
-  if(!dir.exists(outpath)) dir.create(outpath)
-  
+  if (!dir.exists(outpath)) dir.create(outpath)
+
 
   file <- NA
   names(file) <- paste(start.time, "-", stop.time)
-  
-  #Changing the name of the files, so it would contain the name of the hour as well.
-  file <- paste0(outpath, "/", prefix, ".",
-                 paste0(start.time%>% as.character() %>% gsub(' ',"_",.),
-                        "-",
-                        stop.time%>% as.character() %>% gsub(' ',"_",.)), ".clim")
-  
-  if(file.exists(file) & !overwrite){
+
+  # Changing the name of the files, so it would contain the name of the hour as well.
+  file <- paste0(
+    outpath, "/", prefix, ".",
+    paste0(
+      start.time %>% as.character() %>% gsub(" ", "_", .),
+      "-",
+      stop.time %>% as.character() %>% gsub(" ", "_", .)
+    ), ".clim"
+  )
```

Comment:
revert

Reply:
Or rewrite... this one's gonna be hard to read however we indent it!